params/config.go

```diff
@@ -830,8 +830,8 @@ func (c *ChainConfig) Rules(num *big.Int, isMerge bool, timestamp uint64) Rules
 		IsPetersburg:     c.IsPetersburg(num),
 		IsIstanbul:       c.IsIstanbul(num),
 		IsBerlin:         c.IsBerlin(num),
-		IsEIP2929:        c.IsBerlin(num) && !c.IsPrague(num, timestamp),
-		IsEIP4762:        c.IsPrague(num, timestamp),
+		IsEIP2929:        c.IsBerlin(num), // && !c.IsPrague(num, timestamp),
+		IsEIP4762:        false,           //c.IsPrague(num, timestamp),
```

Comment:
this is basically what's new: 2929 still needs to remain active, if we want to have the same gas costs.

When I'll start comparing the gas costs, I will activate 4762 and deactivate 2929 (and the check for gas consumption, of course). This means that some txs will fail when they shouldn't have (and vice-versa) but we would be able to figure it out and only take the ones that match into account.